includes/class-sensei-course-participants.php

```diff
@@ -158,11 +173,13 @@ private function exclude_completed_participants( $post_id ) {
 	}
 
 	/**
-	 * Get the number of learners taking the current course
+	 * Get the number of learners taking the current course.
 	 *
-	 * @since  1.0.0
+	 * @since 1.0.0
+	 * @since 2.0.1 Changed to count enrolled users instead of learners in progress.
```

Comment:
Will we launch it in the `2.0.1`? Or will we create a new major version for all plugins?

Reply:
I think this is at least a 2.1.0, but I'll defer to @donnapep.

Reply:
Since we're not adding any new functionality here, 2.0.1 is fine with me.